tests/test_operators.py

```diff
@@ -309,6 +310,10 @@ def test_not_equal_to(self):
             "target": "var5",
             "comparator": "var6"
         }).equals(pandas.Series([False, False, True])))
+        self.assertTrue(DataframeType({"value": df}).not_equal_to({
+            "target": "var5",
```

Comment:
did you mean to compare var6 and var7? the var properties are out of order when creating the test data

Reply:
No I mean to compare var5 and var7. I just added a new var named var7 because 1-6 were taken. The order and the var names don't matter

Reply:
^^ @nhaydel this last comment is still pending resolution

Reply:
Sorry, didn't realize what you meant. This is updated and ready for re review. @gcampion-cdisc